wandb/sdk/wandb_init.py

```diff
@@ -345,44 +345,74 @@ def getcaller():
 
 def init(
     job_type: Optional[str] = None,
-    dir=None,
+    dir: Optional[str] = None,
     config: Union[Dict, None] = None,  # TODO(jhr): type is a union for argparse/absl
     project: Optional[str] = None,
     entity: Optional[str] = None,
     reinit: bool = None,
     tags: Optional[List] = None,
-    team: Optional[str] = None,
+    team: Optional[str] = None,  # LB What is this?
     group: Optional[str] = None,
     name: Optional[str] = None,
     notes: Optional[str] = None,
     magic: bool = None,  # TODO(jhr): type is union
-    config_exclude_keys=None,
-    config_include_keys=None,
+    config_exclude_keys: Optional[str] = None,  # LB: Seems unsupported
+    config_include_keys: Optional[str] = None,  # LB: Seems unsupported
     anonymous: Optional[str] = None,
-    disable: bool = None,
-    offline: bool = None,
-    allow_val_change: bool = None,
-    resume=None,
-    force=None,
-    tensorboard=None,  # alias for sync_tensorboard
-    sync_tensorboard=None,
-    monitor_gym=None,
-    id=None,
-    settings: Union[Settings, Dict[str, Any], str, None] = None,
-) -> Run:
-    """Initialize a wandb Run.
+    disable: bool = None,  # LB: What is this
+    offline: bool = None,  # LB: What is this
+    allow_val_change: bool = None,  # LB: What is this?
```

Comment:
By default once a key in `wandb.config` is set, we don't allow you to change it.  This is to prevent someone overwriting a sweep config value.  Setting allow_val_change to True removes this check.

Reply:
I am close to getting rid of that, but not ready yet